PyComPWA.cpp

```diff
@@ -45,17 +45,39 @@ PYBIND11_DECLARE_HOLDER_TYPE(T, std::shared_ptr<T>);
 PYBIND11_MODULE(ui, m) {
   m.doc() = "pycompwa module\n"
             "---------------\n";
-  // ------- Logging
-  // reinitialize the logger with level INFO
+  
+  // -----------------------------------------
+  //      Interface to Core components
+  // -----------------------------------------
+  
+  /// Reinitialize the logger with level INFO and disabled log file.
   ComPWA::Logging("INFO");
   py::class_<ComPWA::Logging, std::shared_ptr<ComPWA::Logging>>(m, "Logging")
       .def(py::init<std::string, std::string>(), "Initialize logging system",
-           py::arg("log_level"), py::arg("filename"))
-      .def(py::init<std::string>(), "Initialize logging system",
-           py::arg("log_level"));
+           py::arg("log_level"), py::arg("filename") = "")
+      .def_property("level", &ComPWA::Logging::getLogLevel,
```

Comment:
What happens here if you set it to an illegal value?

Reply:
Thats upto the C++ code

Reply:
Basically nothing happens

Reply:
True